src/napari/utils/logo.py

```diff
@@ -0,0 +1,56 @@
+import re
+from datetime import date
+from importlib import resources
+from pathlib import Path
+
+
+def available_logos():
```

Comment:
```suggestion
def available_logos()- list[str]:
    """Get a list of available logos based on light version of logos""" 
```
Maybe also cache the output.

Reply:
It's pretty cheap, and what if plugins provide logos (as you suggest above)?

Reply:
If a plugin is provided, then the implementation needs to be changed, and then cache (`functools.lru_cache`) could be removed.